scripts/jenkins/cloud/ansible/roles/list_packages/tasks/list-packages.yml

```diff
@@ -45,6 +45,10 @@
   loop: "{{ [cloud_env]+groups['cloud_virt_hosts']|flatten(levels=1) }}"
   loop_control:
     loop_var: item_virtual_hosts
+  # retry used because of rare zypper lock issue
+  retries: 3
+  delay: 3
```

Comment:
Is this long enough? This means that we retry 3 times with a delay of 3 seconds... Which is probably fine in the case where all the repos are provided by the deployer VM, but if there is any chance that there could be any external repos - e.g. IBS repos accessed over Nuremberg link - then this may not be sufficient time...

Reply:
I would that it should be enough but maybe I'm wrong since I have no idea what is going on in there. Anyway because of that it would be nice to know if there is a zypper thing hanging  which could be revealed  specially when 10secs are not enough.